walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,135 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.*; 
+import java.util.Comparator;
+import java.util.Collections;
+import java.util.Iterator;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    
```

Comment:
Refer to bullet 2 from [this whitespace usage section of the style guide](https://google.github.io/styleguide/jsguide.html#formatting-whitespace)

```
A single blank line appears:

1. Between consecutive methods in a class or object literal
  a. Exception: A blank line between two consecutive properties definitions in an object literal (with no other code between them) is optional. Such blank lines are used as needed to create logical groupings of fields.

2. Within method bodies, sparingly to create logical groupings of statements. Blank lines at the start or end of a function body are not allowed.

3. Optionally before the first or after the last method in a class or object literal (neither encouraged nor discouraged).

4. As required by other sections of this document (e.g. 3.6 goog.require and goog.requireType statements).
```

TL;DR no newlines at the start and end of function definitions